### PR TITLE
Added Spanish Language (es_es)

### PR DESCRIPTION
**Pull request by FrannDzs**
Hi, after some time away from Github I corrected the branch and re-did everything from scratch. Cheers!

### CODE REVIEW

File: src/main/resources/assets/immersivepetroleum/lang/es_es.json [hunk 1]
```diff
@@ -118,12 +118,12 @@
 	
 	"_comment": "################## IE-Manual ##################",
 	
-	"manual.immersivepetroleum.main": "Procesado de petróleo",
+	"manual.immersivepetroleum.main": "§lProcesado de petróleo§r",
```
Comment (TwistedGate) [inline src/main/resources/assets/immersivepetroleum/lang/es_es.json:121]: Not a big fan of the category text being **fat**

Any specific reason for that?

File: src/main/resources/assets/immersivepetroleum/lang/es_es.json [hunk 2]
```diff
@@ -118,12 +118,12 @@
...
 	
 	"ie.manual.entry.blank_text": "",
 	
-	"ie.manual.entry.reservoirs.title": "Depósito de fluídos",
-	"ie.manual.entry.reservoirs.subtitle": "Oro negro",
+	"ie.manual.entry.reservoirs.title": "§lDepósito de fluídos§r",
+	"ie.manual.entry.reservoirs.subtitle": "§oOro negro§r",
```
Comment (TwistedGate) [inline src/main/resources/assets/immersivepetroleum/lang/es_es.json:126]: Same with these two

File: src/main/resources/assets/immersivepetroleum/manual/es_es/asphalt.txt [hunk 3]
```diff
@@ -1,3 +1,3 @@
-Hormigón de asfalto
-Material para carreteras
-<&asphalt0>Hormigón de asfalto, unidos por el pegajoso betún, es una alternativa de color más oscuro al Hormigón normal. Puede producirse utilizando los subproductos del procesamiento del petróleo crudo en la <link;immersivepetroleum:distillationtower;§n§oTorre de §n§oDestilación§r>.
+§lHormigón de asfalto§r
+§oMaterial para carreteras§r
```
Comment (TwistedGate) [inline src/main/resources/assets/immersivepetroleum/manual/es_es/asphalt.txt]: And these..

File: src/main/resources/assets/immersivepetroleum/lang/es_es.json (hunk 1 above)
Comment (FrannDzs) [reply src/main/resources/assets/immersivepetroleum/lang/es_es.json:121]: Hello @TwistedGate , yes, I am in responsible for the Spanish translation of immersive engineering, and the entries in the manual have been redone in bold, and to match the rest is why I put it here too, although it is not finished yet...

Comment (FrannDzs) [reply src/main/resources/assets/immersivepetroleum/lang/es_es.json:121]: @TwistedGate Should I fix it or can I keep it that way?

Comment (TwistedGate) [reply src/main/resources/assets/immersivepetroleum/lang/es_es.json:121]: > 
> 
> Hello @TwistedGate , yes, I am in responsible for the Spanish translation of immersive engineering, and the entries in the manual have been redone in bold, and to match the rest is why I put it here too, although it is not finished yet...

Alrighty

> 
> 
> @TwistedGate Should I fix it or can I keep it that way?

Keep it

**Pull request merged**